pywt/data/_readers.py

```diff
@@ -137,3 +137,43 @@ def ecg():
     fname = os.path.join(os.path.dirname(__file__), 'ecg.npy')
     ecg = np.load(fname)
     return ecg
+
+
+def nino():
+    """
+    NINO3 sea surface temperature 1871-1996 (in C; from the UKMO GISST2.3):
```

Comment:
It would be nice to use the nino stuff if possible. Torrence and Compo is the canonical cwt paper. I don't know much about this, but I wonder if example usage is ok (does fair use apply to small amounts of data?).

@pelson: maybe you can help us here - can you comment on whether UKMO has precedent for licensing out small amounts of data for example use in open source? Or is it a bit tricky?